### ISSUE TITLE
HX711 absolute weight computation

### ISSUE THREAD

**Issue by ThomDietrich** [arendst/Tasmota#13981]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

Hey friends,
this is a feature request.

Related to https://github.com/arendst/Tasmota/blob/development/tasmota/xsns_34_hx711.ino

The HX711 weight sensor is supported prominently with two outputs:

- an absolute raw weight value across reboots
- a kg weight value with implicit auto-tare functionality.

However there is no output for:

- an absolute kg weight value across reboots

The last one seems like the one most users actually need.
I would like to understand whether there is a specific reason for this divide. I am not sure which user actually needs an auto-tare sensor and the PR  for the raw value (https://github.com/arendst/Tasmota/pull/7171 by @tobox) even mentions that "to do calibration/tare in upper layers." I can confirm that this indeed works well. I have a simple conversion (`y = ax +b`) doing this in Home Assistant, however would prefer my DIY scale to do this directly. ESPhome supports the conversion directly.

**TL;DR;** Would it be possible to change or extend the code for the HX711 sensor, so raw value conversion and reporting of absolute kg measurements is directly handled by Tasmota? If needed I could try to contribute a PR.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Sonoff Basic
- [x] Tasmota binary firmware version number used: 10.0.0
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: Webtool
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
```
- [ ] Provide the output of this command: `Status 0`:
```lua
```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by ascillato2** [comment]
Moving to discussion as this is a feature request and not a Bug Report.
